include/aie/Dialect/AIEVec/IR/AIEVecOps.td

```diff
@@ -117,7 +117,7 @@ def AIEVec_FMAOp :
   AIEVec_Op<"mac", [
     NoSideEffect
   ]>,
-  Arguments<(ins AnyVector:$lhs, AnyVector:$rhs, AIEVec_Acc:$acc,
+  Arguments<(ins AnyVector:$lhs, AnyVector:$rhs, AnyTypeOf<[AIEVec_Acc, VectorOf<[F32]>]>:$acc,
```

Comment:
I think we can just remove AIEVec_Acc

Reply:
For the sake of keeping the scope of this PR to removing the AIEVec_Acc (and thus the SRS and UPS) from just the float path, I opted to keep the AIEVec_Acc to reflect the current state of how the rest of the integer operations still use it to model the accumulator. AIEVec_Acc is still needed here due to how UPS is specified to have an AIEVec_Acc result: removing the AIEVec_Acc here would require additional changes to UPS that are more inline with the PR for #101 that I'm going to make that removes AIEVec_Acc entirely.